services/keycloak/startup-scripts/00-configure-lagoon.sh

```diff
@@ -1811,6 +1811,15 @@ function configure_keycloak {
     configure_token_exchange
     update_add_env_var_to_project
 
+    # Here we source in any extra configuration
+    echo "Checking for any custom startup scripts"
+    if ls /configure-lagoon-extra/*.sh 1> /dev/null 2>&1; then
```

Comment:
Instead of this logic you could do something like:
```bash
     for FILE in /configure-lagoon-extra/*.sh; do
        [ -r "$FILE" ] || continue
        echo "Found $FILE - executing in current shell"
        source "$FILE";
      done
```

Reply:
Thanks @smlx!